src/mumble/AudioOutput.cpp

```diff
@@ -3,6 +3,14 @@
 // that can be found in the LICENSE file at the root of the
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
+// <cmath> includes <math.h>, but only if it isn't already included.
+// We include <cmath> as first header to make sure that we include <math.h> with _USE_MATH_DEFINES.
```

Comment:
What for? What issue are we solving here?

Reply:
I improved the comment.